third_party/openvino/ienn/src/ie_model.cpp

```diff
@@ -504,6 +504,66 @@ ie_operand_t* Model::AddReduceMean(ie_operand_t* input,
   return CreateOperand(node_name);
 }
 
+ie_operand_t* Model::AddResample(ie_operand_t* input,
+                                 ie_resample_options_t* options) {
+  auto input_node = name_node_map_[input->name];
+  auto input_shape = input_node.get_shape();
+  auto axes_node = op::Constant::create(element::i64, Shape{4}, {0, 1, 2, 3});
+  SizeVector scales;
+  scales.reserve(4);
+  if (options->scalesCount == 0) {
+    for (uint32_t i = 0; i < 4; ++i) {
+      scales.push_back(static_cast<float>(options->sizes[i]) /
```

Comment:
What's the behavor if both of `sizes `and `scales `are empty?

Reply:
Either `sizes` or `scales` is empty. The `sizes` and `scales` can't both be empty.

Reply:
```
dictionary MLResampleOptions {
  MLInterpolationMode mode = "nearest-neighbor";
  sequence<float> scales;
  sequence<long> sizes;
};
MLOperand resample(MLOperand input, optional MLResampleOptions options = {});
```
They are host in the option, both of them can be empty. Does it mean the api should be defined like below if `scales `is required? `sizes `is optional  because the options.scales argument is ignored when the target sizes are specified. @huningxin 
```
dictionary MLResampleOptions {
  MLInterpolationMode mode = "nearest-neighbor";
  sequence<long> sizes;
};
MLOperand resample(MLOperand input, sequence<float> scales, optional MLResampleOptions options = {});
```

Reply:
> What's the behavor if both of `sizes `and `scales `are empty?

I think it should be treated as scales as [1.0, 1.0, 1.0, 1.0]. Please file an issue to webnn spec repo, we can fix adding the default value.

Reply:
> Does it mean the api should be defined like below if `scales `is required?

This also sounds a solution. Please file the issue and we can discuss.

Reply:
As polyfill implementation, it throws if neither scales nor sizes is defined. Anyway, we need to discuss and define the correct behavior in the spec.